demo/app/scripts/controllers/main.js

```diff
@@ -204,6 +204,12 @@ angular.module('angularGanttDemoApp')
                         api.tasks.on.resizeEnd($scope, addEventName('tasks.on.resizeEnd', logTaskEvent));
                     }
 
+                    if (api.tasks.on.drawBegin) {
+                        api.tasks.on.drawBegin($scope, addEventName('tasks.on.drawBegin', logTaskEvent));
+                        //api.tasks.on.draw($scope, addEventName('tasks.on.draw', logTaskEvent));
```

Comment:
It's commented out just like the resize and move events of the movable plugin. Why are those commented out?